### PR TITLE
Book store UI

### PR DESCRIPTION
**Pull request by henask12**
### Description:
This pull request addresses the task of adding any missing JSX elements to the project while ensuring the final design aligns with Zeplin's specifications. The goal is to achieve an exact match, including all required elements and their respective styling as per the provided design.

To accomplish this, I have integrated Tailwind CSS, a utility-first CSS framework, to facilitate the styling process and maintain consistency throughout the website.

### Key changes in this pull request:

- [x]  **Addition of Missing JSX Elements:** During the development process, I have carefully analyzed Zeplin's design to identify any missing JSX elements that were not present in the original codebase. These missing elements have been created and incorporated into the project.

- [x] **Styling to Match Zeplin Design:** Utilizing the detailed CSS rules provided in Zeplin's design, I have applied the necessary styles to all newly added JSX elements, as well as existing ones. This ensures that the final presentation closely resembles the Zeplin design, adhering to the specified styles, colors, typography, and spacing.

- [x] **Integration of Tailwind CSS:** To streamline the styling process and maintain a scalable and organized CSS codebase, I have integrated Tailwind CSS. This powerful framework allows for efficient styling through utility classes, enabling faster development while adhering to the design specifications.

- [x] **Testing and Verification:** I have thoroughly tested the changes across multiple devices and browsers to ensure cross-compatibility and responsiveness. Additionally, I have verified that the website now exhibits the exact appearance intended by Zeplin's design.

- [x] **Code Quality and Maintainability:** While making these additions and modifications, I have maintained the code's readability, adhered to coding best practices, and provided comments where necessary for future reference.

Please review this pull request to ensure that all the required elements and styles match Zeplin's design accurately. Any feedback, suggestions, or improvements are more than welcome, and I am committed to addressing them promptly.

Thank you for considering this contribution to the project!

### CODE REVIEW

File: src/components/booklist.css
Comment (joskalenda) [inline src/components/booklist.css:1]: - [ ] Your design is clean, however, I would suggest some changes referring to the given template
- You should better use this  `font-family: Montserrat;` as per the original template
- Your add button should be aligned closer to the category button, not below 


YOUR DESIGN

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @henask12 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Green linters
- [ ].. ✅ Readme file
- [ ].. ✅ Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/booklist.css
Comment (henask12) [reply src/components/booklist.css:1]: hey @joskalenda  what is the device width 
image
image
i don't see it from my end

Comment (Lucas-Masaba) [reply src/components/booklist.css:1]: - [ ] Please make sure that your design matches the zeplin design as mentioned in the project requirements. That means you should follow the layout(where the buttons and other items should be positioned) and fonts(colors, text etc). 
### Your Implementation
image

### Required
image

As you can see, yours looks different from the original design. Please make sure it looks similar.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @henask12  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost-there

 ## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navigation.js [hunk 1]
```diff
@@ -7,28 +7,71 @@ import Categories from './Categories';
...
-      <div className="container mx-auto px-4 flex items-center justify-between">
-        <div className="logo">
-          <span className="text-white font-bold text-xl">Book Store</span>
+    <nav className="bg-white-800 py-4 border border-white-100 rounded-lg shadow-sm">
+      <div className="container mx-auto px-4 sm:px-8 md:px-16 lg:px-20 xl:px-24 2xl:px-32 flex flex-col sm:flex-row justify-between items-center">
+        <div className="logo flex items-center">
+          <span
+            className="text-blue-500 font-bold text-xl hidden sm:inline-block sm:text-xl"
+            style={{
+              fontFamily: 'Montserrat, sans-serif',
```
Comment (henask12) [inline src/components/Navigation.js:16]: i don't know if you checked my code or not i have used it and you can see here as an example @Lucas-Masaba

**Review by algerina — APPROVED** [review]
Hi @henask12  ,

Great job so far!  

May I highlight the following: 
- [x] Your  have a descriptive PR and commit history👏  ✔️ 
- [x] Your project looks great✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**